src/main/java/org/zalando/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -108,6 +109,7 @@ public Properties getKafkaProducerProperties() {
         producerProps.put("request.timeout.ms", kafkaSettings.getRequestTimeoutMs());
         producerProps.put("batch.size", kafkaSettings.getBatchSize());
         producerProps.put("linger.ms", kafkaSettings.getLingerMs());
+        producerProps.put(ProducerConfig.COMPRESSION_TYPE_CONFIG, "lz4");
```

Comment:
It's great that you use this constant instead of a String like the few lines above. Do you think you could do the same for the other properties?